testdata/p4_16_errors_outputs/issue306.p4-stderr

```diff
@@ -1,36 +1,6 @@
-issue306.p4(30): error: Structure struct my_packet does not have a field stack
+issue306.p4(30): [--Werror=type-error] error: Structure struct my_packet does not have a field stack
```

Comment:
It looks like for this error the two arguments should be swapped, because these are actually not duplicate messages.

Reply:
Attached is how the errors look if I switch it. It is true that it identifies all uses of the .stack member, but it does seem a bit redundant? Let me know if you prefer it this way.
[issue306.p4-stderr.txt](https://github.com/p4lang/p4c/files/2829218/issue306.p4-stderr.txt)

Reply:
I think it's better. We could avoid printing the struct position altogether, although it's probably useful the first time.